client/src/main/scala/drt/client/components/Navbar.scala

```diff
@@ -100,22 +100,19 @@ object Navbar {
                   navbarModel.featureGuideViewIds.renderReady { userFeatureViewCount =>
                     <.div(^.className := "main-menu-items",
                       <.div(if (trainingDataTemplates.nonEmpty) {
-                        <.a(^.onClick ==> {
-                          handleOpenDialog
-                        }, MuiBadge(badgeContent = {
-                          val badgeCount = calculateBadgeCount(userFeatureViewCount, trainingDataTemplates.map(_.id.map(_.toString).getOrElse("0")))
-                          <.span(badgeCount)
-                        }, showZero = false, color = "primary")("New Feature"))
+                        val badgeCount = calculateBadgeCount(userFeatureViewCount, trainingDataTemplates.map(_.id.map(_.toString).getOrElse("0")))
+                        <.a(MuiBadge(badgeContent = <.span(badgeCount),
+                          invisible = {
+                            badgeCount == 0
+                          }, color = "primary")(Icon.laptopLg), <.span(if (badgeCount != 0) ^.className := "badge-text" else " ", "New Feature"), ^.onClick ==> handleOpenDialog)
```

Comment:
Rather than conditionally adding a class to add a left margin, you can use a mui Box with sx `display: flex, gap: 10px` and the gap only applies if there's more than one element in it